pkg/cmd/roachtest/test_registry.go

```diff
@@ -127,6 +128,25 @@ func (r *testRegistryImpl) prepareSpec(spec *registry.TestSpec) error {
 	}
 	spec.Tags = append(spec.Tags, "owner-"+string(spec.Owner))
 
+	// At the time of writing, we expect the roachtest job to finish within 24h.
+	// Since each individual test may not be scheduled until a few hours in,
+	// individual tests should expect to take "less time". Longer-running
+	// tests require the weekly tag.
```

Comment:
Is it worth mentioning that Teamcity has a 21 hour timeout (per the old comment)?